src/librustc_lint/builtin.rs

```diff
@@ -41,6 +41,7 @@ use lint::{Level, Context, LintPass, LintArray, Lint};
 
 use std::collections::BitSet;
 use std::collections::hash_map::Entry::{Occupied, Vacant};
+#[allow(deprecated)]
 use std::num::SignedInt;
```

Comment:
Shouldn't this be removed? I would figure the inherent methods would be used instead.